src/Websites/Websites/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 * Fixing issue where webapp Tags were getting deleted when migrating App to new ASPwhere webapp Tags were getting deleted when migrating App to new ASP
 * Fixing the Publish-AzureWebapp to work across Linux and windows
 * Update example in `Get-AzWebAppPublishingProfile` reference documentation
+* Adding cmdlets to work with Access Restrictions
```

Comment:
Please include the new cmdlet names

Reply:
Thank you @markcowl , I have updated the change log description.